### PR TITLE
Add redeemEnableTime for lock4reputation schemes

### PR DESCRIPTION
**Pull request by orenyodfat**
use infra 09

auction4reputation - add transferToWallet

### CODE REVIEW

File: contracts/schemes/Auction4Reputation.sol [hunk 1]
```diff
@@ -84,7 +90,7 @@ contract Auction4Reputation is Ownable {
      */
     function redeem(address _beneficiary, uint _auctionId) public returns(bool) {
         // solium-disable-next-line security/no-block-members
-        require(now > auctionsEndTime, "check the auctions period pass");
+        require(now > redeemEnableTime, "now > redeemEnableTime");
```
Comment (dkent600) [inline contracts/schemes/Auction4Reputation.sol]: Should be >=

File: contracts/schemes/Locking4Reputation.sol [hunk 2]
```diff
@@ -46,7 +47,7 @@ contract Locking4Reputation {
      */
     function redeem(address _beneficiary) public returns(bool) {
         // solium-disable-next-line security/no-block-members
-        require(block.timestamp > lockingEndTime, "check the lock period pass");
+        require(block.timestamp > redeemEnableTime, "now > redeemEnableTime");
```
Comment (dkent600) [inline contracts/schemes/Locking4Reputation.sol]: Should be >=

File: contracts/schemes/Auction4Reputation.sol [hunk 3]
```diff
@@ -128,4 +134,15 @@ contract Auction4Reputation is Ownable {
         return auctions[_auctionId].bids[_bidder];
     }
 
+    /**
+     * @dev transferToWallet transfer the tokens to the wallet.
+     *      can be called only after auctionsEndTime
+     */
+    function transferToWallet() public {
+      // solium-disable-next-line security/no-block-members
+        require(now > auctionsEndTime, "now > redeemEnableTime");
```
Comment (leviadam) [inline contracts/schemes/Auction4Reputation.sol]: Error message is wrong.

Comment (orenyodfat) [reply contracts/schemes/Auction4Reputation.sol]: thanks @leviadam 
fixed at 247ef62

**Pull request merged**